siliconcompiler/targets/fpga_nextpnr_flow_demo.py

```diff
@@ -11,24 +11,24 @@ def make_docs(chip):
 ####################################################
 # Target Setup
 ####################################################
-def setup(chip):
+def setup(chip, toolflow='vpr'):
```

Comment:
you don't use `toolflow`, please return this back to `fpga_flow_demo` and use that flag to indicate which flow you are loading